ui/app/src/components/ItemDisplay/ItemDisplay.tsx

```diff
@@ -81,7 +81,7 @@ const ItemDisplay = forwardRef<HTMLSpanElement, ItemDisplayProps>((props, ref) =
 		// Prevents crashing if the item is nullish
 		return null;
 	}
-	const isDisabledItem = item.stateMap.disabled;
+	const isDisabledItem = item.stateMap?.disabled;
```

Comment:
Validating stateMap of an item since ItemDisplay is used in some places to render LightItems (they don't have stateMap)